controllers/gatekeeper_controller_test.go

```diff
@@ -48,6 +48,59 @@ func TestDeployValidatingWebhookConfig(t *testing.T) {
 	g.Expect(getStaticAssets(gatekeeper)).NotTo(ContainElement(ValidatingWebhookConfiguration))
 }
 
+func TestCustomNamespace(t *testing.T) {
+	g := NewWithT(t)
+	namespace := "testns"
+	gatekeeper := &operatorv1alpha1.Gatekeeper{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: namespace,
+		},
+	}
+	// Rolebinding namespace overrides
+	rolebindingManifest, err := util.GetManifest(RoleBindingFile)
+	g.Expect(err).ToNot(HaveOccurred())
+	g.Expect(rolebindingManifest).ToNot(BeNil())
+	err = crOverrides(gatekeeper, RoleBindingFile, rolebindingManifest)
+	g.Expect(err).ToNot(HaveOccurred())
+	subjects, found, err := unstructured.NestedSlice(rolebindingManifest.Obj.Object, "subjects")
```

Comment:
Do you think we should check subjects is not `nil`?

Reply:
I don't think it can be the case. Subjects is an array of type Subject but we can just add it.